api/src/main/java/com/velocitypowered/api/util/ProtocolConstants.java

```diff
@@ -0,0 +1,100 @@
+package com.velocitypowered.api.util;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ProtocolConstants {
```

Comment:
Rename to `ProtocolVersion`.

Reply:
What to do about the name conflict with `ProtocolVersion` in `StateRegistry`?